frontend/packages/api_client/src/apis/V2Api.ts

```diff
@@ -92,6 +95,11 @@ export interface InstancesOpenedxConfigReadRequest {
     id: string;
 }
 
+export interface InstancesOpenedxConfigThemeConfigRequest {
```

Comment:
We have control, but at the expense of adding override decorators to everything. 
Since it would make the backend side way less flexible we're not using it.

Reply:
@giovannicimolin, nit: if you are going to override the name, can you name it `InstancesOpenEdXConfigThemeConfigRequest` to ensure that camel case is used throughout and `Open edX` is spelled the same as it is done in the backend code? This is not required if you are not going to add the overrides.

Reply:
@lgp171188 I ended up not adding a decorator to change the name of this variable. I prefer to not mess around much with the auto-generated code for now.